my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
or just `import matplotlib.pyplot`  (preferred by others in our group)

or `import matplotlib.pyplot as plt` (my favorite alias)